library/std/arrays.qs

```diff
@@ -647,8 +647,11 @@ namespace Microsoft.Quantum.Arrays {
         for index in 0..interleavedLength - 1 {
             let originalIndex = index / 2;
             let value =
-                if index % 2 == 0 {first[originalIndex]}
-                else {second[originalIndex]};
+                if index % 2 == 0 {
+                first[originalIndex]
```

Comment:
Just browsing these files, and this doesn't look right. I assume it should be indented from the if statement above it?

Reply:
Indentation logic will be addressed by a future PR.